js/views/settingsVw.js

```diff
@@ -91,6 +114,15 @@ module.exports = Backbone.View.extend({
     this.moderatorCount = 0;
 
     this.fetchModel();
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
+      var char = String.fromCharCode(e.which).toLowerCase(),
+          ctrl = (e.ctrlKey || e.metaKey) && !e.altKey;
+
+      if (ctrl && char == config.keyShortcuts.save) {
+        window.obEventBus.trigger('saveCurrentForm', self._state);
```

Comment:
Oh, I see.

@mariodian my advice was the global event should only be window.obEventBus.trigger('saveCurrentForm'), and each event should decide for itself what it does with that.

Why are you passing the view around?